packages/frontend/core/src/modules/share-menu/view/share-menu/copy-link-button.tsx

```diff
@@ -70,6 +87,7 @@ export const CopyLinkButton = ({
           <span
             className={clsx(styles.copyLinkShortcutStyle, {
               secondary: secondary,
+              hidden: containerWidth < 320,
```

Comment:
in which case the width of menu will change?

Reply:
When the width of the doc page is below a certain level, the share button will be received in the three-dot menu of the header, and the share menu will open from the submenu of the three-dot menu. 
If the client width is insufficient to display the full width of the share menu at this time, it will scale down, as shown in the video example in the description.